static/js/featuredialog.js

```diff
@@ -0,0 +1,37 @@
+var FeatureDialog = Backbone.View.extend({
+	className: "feature-dialog",
+    events: {
+		"submit" : "submit",
+		"click .save-button" : "submit",
+		"click .close-button" : "close"
+	},
+	initialize: function(options) {
+		this.markers = options.markers;
+		this.type = options.type;
+		this.event = options.event;
+
+	},
+	render: function() {
+		this.$el.append($("#feature-dialog").html());
```

Comment:
In *this* line it should be `feature-dialog-template`...
The template contains the modal, so when you take the template's content you take the modal.
But the `id`s below are good.